internal/models/network.go

```diff
@@ -69,7 +69,8 @@ type SignalData struct {
 
 type ErrorsData struct {
 	CommonData
-	Device Device `json:"device,omitempty"`
+	Device  Device `json:"device,omitempty"`
+	TokenID uint64 `json:"vehicleTokenID"`
```

Comment:
```suggestion
	VehicleTokenID uint64 `json:"vehicleTokenId"`
```

Reply:
we have similar in here - https://github.com/DIMO-Network/edge-network/blob/c5a3f45ef4317ef3fdf46e0fa6b725dc2adffd88/internal/models/network.go#L139-L138

Reply:
updated